apps/web/src/pages/Home/IAPManagerHomePage/IAPManagerHomePage.tsx

```diff
@@ -0,0 +1,871 @@
+                  style={{ transform: "rotate(90deg) translate(20%, -25%)" }}
+                  data-h2-width="base(100%)"
+                  data-h2-color="base(primary.darker)"
+                />
+                <img
+                  src={section5Img}
+                  alt={intl.formatMessage({
+                    defaultMessage:
+                      "Indigenous woman taking a break from her program delivery work on a laptop.",
+                    id: "EtOnRo",
+                    description:
+                      "Description of a decorative image of a woman and a laptop",
+                  })}
+                  data-h2-min-height="base(32rem) p-tablet(initial)"
+                  data-h2-height="base(100%)"
+                  data-h2-width="base(100%)"
+                  width="451px"
+                  height="523px"
+                  data-h2-display="base(block)"
+                  data-h2-position="base(relative)"
+                  data-h2-margin="base(0 auto)"
+                  style={{
+                    objectFit: "cover",
+                  }}
+                />
+                <img
+                  src={section5FlourishImg}
+                  alt={intl.formatMessage({
+                    defaultMessage:
+                      "Indigenous talking stick, which represents respect in many Indigenous communities.",
+                    id: "+tGYjC",
+                    description:
+                      "Description of a decorative image of a talking stick",
+                  })}
+                  data-h2-position="base(absolute)"
+                  data-h2-location="base(auto, auto, -x8, -x1)  p-tablet(auto, auto,-x4, -x1) l-tablet(auto, auto,-x8, -x1)"
+                  data-h2-width="base(80%) base(150%)"
+                />
+              </div>
+            </div>
+            <div
+              data-h2-flex-item="base(1of1) p-tablet(4of7)"
+              data-h2-position="base(relative)"
+            >
+              <Heading
+                size="h3"
+                data-h2-font-weight="base(700)"
+                data-h2-margin="base(x5, 0, x2, 0) p-tablet(0, 0, 0, 0)"
+                data-h2-color="base(primary.darker)"
+                data-h2-text-align="base(center) p-tablet(left)"
+              >
+                {intl.formatMessage({
+                  defaultMessage: "A commitment to diverse digital talent",
+                  id: "29o8/W",
+                  description:
+                    "Title of the 'A commitment to diverse digital talent' section",
+                })}
+              </Heading>
+              <p data-h2-margin="base(x1, 0, x1, 0)">
+                {intl.formatMessage({
+                  defaultMessage:
+                    "The Digital Ambition, released in 2022, provides direction on how to increase representation of under-represented groups by leveraging programs like the IT Apprenticeship Program for Indigenous peoples.",
+                  id: "eSzizR",
```

Comment:
I just noticed the underlining in the Figma file, is there supposed to be a link to something here?

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/3435ac42-8077-4a84-b1aa-03c21469aac3)

Reply:
@gobyrne Do you have the link for this?

Reply:
EN: https://www.canada.ca/en/government/system/digital-government/digital-ambition.html
FR: https://www.canada.ca/fr/gouvernement/systeme/gouvernement-numerique/ambition-numerique.html